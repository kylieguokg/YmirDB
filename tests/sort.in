sort a
set a
get a
sort a
get a
set a 1
get a
sort a
get a
set a 90 -1000 -20 1000 28 0
get a
sort a
get a
set repeat 28 28 28 28 28 28
get repeat
sort repeat
get repeat
bye
