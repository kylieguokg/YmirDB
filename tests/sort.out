> no such key

> ok

> []

> ok

> []

> ok

> [1]

> ok

> [1]

> ok

> [90 -1000 -20 1000 28 0]

> ok

> [-1000 -20 0 28 90 1000]

> ok

> [28 28 28 28 28 28]

> ok

> [28 28 28 28 28 28]

> bye
